src/helpers/aws_helper.py

```diff
@@ -1846,3 +1848,47 @@ def execute_commands_on_ec2_by_tags_and_wait(
 
     console_printer.print_info(f"Response from ssm {resp}")
     time.sleep(timeout)
+
+
+def purge_sqs_queue(queue_name, aws_region="eu-west-2"):
+    console_printer.print_info(f"Purging queue: {queue_name}")
+    service_name = "sqs"
+    client = boto3.client(service_name, region_name=aws_region)
```

Comment:
You should use the `get_client` method in this helper. It allows us to standardise all boto3 clients. Same for the methods below.

Reply:
Thanks Steve. changes applied.